src/lib/mongoBackend/mongoUpdateContextSubscription.cpp

```diff
@@ -245,8 +245,36 @@ HttpStatusCode mongoUpdateContextSubscription
       /* Build conditions array (including side-effect notifications and threads creation)
        * In order to do so, we have to create and EntityIdVector and AttributeList from sub
        * document, given the processConditionVector() signature */
-       EntityIdVector enV = subToEntityIdVector(sub);
-       AttributeList attrL = subToAttributeList(sub);
+       EntityIdVector enV;
+       AttributeList attrL;
+       if (version == "v1")
+       {
+         enV = subToEntityIdVector(sub);
+         attrL = subToAttributeList(sub);
+       }
+       else // v2
+       {
+         // In v2 entities and attribute are updatable (as part of subject or notification) wo
+         // we have to check in order to know if we get the attribute from the request or from
+         // the subscription
+         if (requestP->entityIdVector.size() > 0)
+         {
+           enV = requestP->entityIdVector;
```

Comment:
I'm not explicitely releasing (as part of this PR) but I guess it could be done as part of the releasing of contaning classes in parts of the code not touched by this PR.

By the moment 295/565 in test harness without any fail...